analysis/parameter_tuning.py

```diff
@@ -153,6 +179,35 @@ def _find_candidates(histogram: histograms.Histogram) -> List:
         max_contributions_per_partition=linf_bounds)
 
 
+def _find_candidates_quantiles(histogram: histograms.Histogram,
+                               n_max: int) -> List[int]:
+    """Implementation of ParametersSearchStrategy.QUANTILES strategy."""
+    quantiles_to_use = [0.9, 0.95, 0.98, 0.99, 0.995]
+    candidates = histogram.quantiles(quantiles_to_use)
+    candidates.append(histogram.max_value)
+    candidates = list(set(candidates))  # remove duplicates
+    candidates.sort()
+    return candidates[:n_max]
```

Comment:
maybe log a warning because such behaviour can signal about a mistake? becuase n_max > 5 is weird value.